salt/environment_settings.yml

```diff
@@ -424,6 +424,12 @@ environments:
           - elasticsearch-production-apps.odl.mit.edu
         security_groups:
           - webapp
+      starcellbio:
+        business_unit: starteam
+        domains:
+          - starcellbio.mit.edu
```

Comment:
Wouldn't using the odl wildcard cert cause mismatch problems for this domain?

Reply:
Good point...